com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -122,6 +125,11 @@ public ulong LocalClientId
         /// </summary>
         public bool IsServer => ConnectionManager.LocalClient.IsServer;
 
+        /// <summary>
+        /// Gets whether or not the current server (local or remote) is a host - i.e., also a client
+        /// </summary>
+        public bool ServerIsHost => ConnectionManager.ConnectedClientIds.Contains(ServerClientId);
```

Comment:
You could just use `ConnectionManager.LocalClient.IsHost; ` here.

Reply:
Only on the server side. This is used client side as well to detect if the server is connected to is a host (which is primarily relevant to figuring out how to route `SendTo.ClientsAndHost` messages, but I thought users might sometimes find this useful as well)

Reply:
Ahhhh... context is always 20:20.